launch-cluster.sh

```diff
@@ -168,6 +168,7 @@ done
 (
   echo EXPECTED_GENESIS_HASH="$GENESIS_HASH"
   echo EXPECTED_SHRED_VERSION="$SHRED_VERSION"
+  echo EXPECTED_BANK_HASH="$8osXYbYF7drjZAJedHuwB8A56t7Pwa6bZbtCjiVhJBbT" # TODO: add ledger-tool command to fetch this correctly...
```

Comment:
Just a stub for now.  We don't need this until somebody wants to next launch a test cluster with these scripts...

Reply:
`$` is a typo here, no?

Reply:
yep, thanks